exec/java-exec/src/test/java/org/apache/drill/TestParquetMissingColumns.java

```diff
@@ -0,0 +1,91 @@
+package org.apache.drill;
+
+import org.apache.drill.common.types.TypeProtos;
+import org.apache.drill.common.types.Types;
+import org.apache.drill.exec.record.BatchSchema;
+import org.apache.drill.exec.record.BatchSchemaBuilder;
+import org.apache.drill.exec.record.metadata.SchemaBuilder;
+import org.apache.drill.test.ClusterFixture;
+import org.apache.drill.test.ClusterTest;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * Covers selecting completely missing columns from a parquet table. Should create Nullable Int
+ * ValueVector in that case since there is no chance to guess the correct data type here.
+ */
```

Comment:
Note that `NULLABLE INT` is just a guess; there is nothing special about it. EVF allows the reader to define its own "default" column type. For example, for CSV, columns are NEVER `INT`, so it is better to guess `VARCHAR`, which is the only type CSV supports. Parquet can play the same trick if doing so is useful.

Reply:
`NULLABLE INT` is just what was there when I came to this issue. I think it would be easy to make it configurable by user (if there is any function able to parse type from string). Should we add it?